docs/monthly-meeting/2023-09.md

```diff
@@ -0,0 +1,118 @@
+- [**Meeting reports**](https://docs-community.readthedocs.io/en/latest/monthly-meeting/index.html) (the latest one might be an [**unmerged PR**](https://github.com/python/docs-community/pulls))
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername` / Discord if different)
+- CAM Gerlach / `@CAM-Gerlach`
+- Daniele / `@EvilDMP`
+- Petr Viktorin / `@encukou`
+- Adam Turner / `@AA-Turner`
+- Hugo van Kemenade / `@hugovk`
+- Maciej Olko / `@m-aciek` / `@m_aciek`
+- Ryan / `@ryan-duve`
+- Marcus Sherman / `@betteridiot`
+- Erlend / `@erlend-aasland`
+- Ege Akman / `@egeakman`
+- Ezio Melotti / `@ezio-melotti`
+
+
+## Introductions
+
+- Maciej, from Poland, works on Polish translation
+
+
+## Reports and celebrations
+
+- [Hugo] [docs.python.org](https://docs.python.org) now has a dark theme! 🌒
+
+- [Hugo] Little things can be big things: I added a shortcut at the top of the [datetime](https://docs.python.org/3/library/datetime.html) docs to take you directly to the percent formatting codes. [People are happy!](https://mastodon.social/@hugovk/110951423574144386)
+
+- [Adam] Six releases of [Sphinx](https://www.sphinx-doc.org/en/master/changes.html)!
+
+- [Adam] New PEP helper & validator ([`check-peps.py`](https://github.com/python/peps/commit/814ceede975056cc56a019bbed62b29a6a875830))
+
+- [Adam] [Datamodel headings](https://github.com/python/cpython/pull/108146) for readability
+
+- [Adam] Work to [remove references to Distutils](https://github.com/python/cpython/issues/92584) in the documentation ahead of the 3.12.0 release
+
+
+## Discussion
+
+- [Hugo] python-docs-theme:
+  - Need a release of 2023.8 to fix some dark theme bugs: [python/python-docs-theme#152](https://github.com/python/python-docs-theme/issues/152)
+  - Have a PR ready to release using Trusted Publishers, need someone to set it up on PyPI: [python/python-docs-theme#148](https://github.com/python/python-docs-theme/pull/148)
+  - These are both waiting on people who can make releases. Adam: should we review the group of people who can make releases, and look to expand it? Potentially ties in to a [Python organisation](https://github.com/python/steering-council/issues/207) on PyPI.
+
+
+- [Ege/Hugo/CAM] Updates and discussion on Plausible hosting:
+  - PSF folks opposed to self-hosting
+  - Paid hosting is a *little* expensive
+  - We got a 15% discount from them, which would be \$1400-\$1500 yearly
```

Comment:
Are these public numbers? (If not, too late, but perhaps we should avoid putting in a public website; a GH PR is at least a little less discoverable)

Reply:
Oof, should we redact the numbers? Even if we do redact the numbers it'll still be visible in the commit history.

Reply:
Agreed to keep it